src/translations/translations_NO.json

```diff
@@ -126,6 +126,10 @@
     "tags": "Emneord"
   },
   "delivered_by": "Levert av",
+  "embed": {
+    "copy_embed_code": "Kopier innebyggingskode",
```

Comment:
burde det vært en forklaring på at det er main-content man bygger inn -  og ikke f.ex ressurs-siden med metadata , eller noen av de andre innholds-filene?

Reply:
kanskje bare legge på  "...for hovedinnhold" på overskriften ?

Reply:
Fikset